hw1103/src/main/java/com/example/hw1103/MainActivity.java

```diff
@@ -0,0 +1,107 @@
+package com.example.hw1103;
+
+import android.os.Bundle;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+
+import com.example.hw1103.backend.ColorManager;
+import com.example.hw1103.backend.User;
+import com.example.hw1103.fragments.FeedFragment;
+import com.example.hw1103.fragments.TrainingRecordFragment;
+
+import java.util.Objects;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final boolean TRUE = true;
+    public static final int NULL = 0;
```

Comment:
Null should be null, not zero, please rename.

Reply:
done